src/main/java/org/zalando/nakadi/plugin/api/authz/EventTypeAuthz.java

```diff
@@ -0,0 +1,9 @@
+package org.zalando.nakadi.plugin.api.authz;
+
+public interface EventTypeAuthz
+{
```

Comment:
let's try to be more consistent with the formatting. Most `{` don't go in the new line. Also, regarding blank lines, we don't have strict rule for it, but usually we don't leave blank lines in between the last method definition and the closing `}`.